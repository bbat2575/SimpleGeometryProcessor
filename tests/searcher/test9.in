-17.000, 10.000
-20.0000, 11.0000
32.500, -49.13
48.930, -36.13
7.33, 45.8900000000000000000
9.0, 0.000
-0.000, 38.37
1.0, 2.0
3.00, 4.00