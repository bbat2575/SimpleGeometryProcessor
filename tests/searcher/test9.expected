read 2 points
1.00, 2.00
3.00, 4.00
This is not a triangle
